spring-cloud-stream-schema/src/main/java/org/springframework/cloud/stream/schema/avro/AvroMessageConverterProperties.java

```diff
@@ -32,6 +33,8 @@
 
 	private Resource[] schemaLocations;
 
+	private Resource[] schemaImports;
```

Comment:
I am confused. Why do we need this? Isn't it the same as `schemaLocations`?
```
"--spring.cloud.stream.schema.avro.schema-locations=classpath:schemas/Command.avsc",
"--spring.cloud.stream.schema.avro.schema-imports=classpath:schemas/imports/Sms.avsc, classpath:schemas/imports/Email.avsc, classpath:schemas/imports/PushNotification.avsc" };
```

Reply:
Imports should be parsed before Locations, you can have a look at maven avro plugin as a reference https://github.com/apache/avro/blob/master/lang/java/maven-plugin/src/main/java/org/apache/avro/mojo/AbstractAvroMojo.java#L162

Reply:
When you have dependent schemas like the one in the tests (command requires sms,email,push noticlfication) schema should first load the dependencies so that it can parse the root. If you remove imports and change the ordering when you do clean install you should not be able to generate classes. Let's assume you generated classes somehow even If you try this example when you use schema registry client you will see that it throws exception

Reply:
Ok, fair enough. So would you be able to provide a javadoc-like short sentence for both `schemaLocations` and `schemaImports`, so I can include them as part of the polishing/merging. This will help further to avoid the confusion.

Reply:
@olegz okay done, please check it out now